new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+# yum update --skip-broken -y
+
+# install requirements for python-lxml
+yum install python-devel gcc-c++ gcc libxslt-devel libxml2-devel python-argparse zlib-devel -y
```

Comment:
@ran-z, @0lvin  I assume that we know this will not be production ready. If we want to be able to provide this in an offline method, we should compile whatever we need and create an rpm.